test/release/examples/primers/chplvis/chplvis.html

```diff
@@ -0,0 +1,312 @@
+</ul>
+<h4>Display Interaction</h4>
+<p>
+Clicking on elements of the display will bring up more information.
+Clicking on a locale will open a window for that locale showing
+information for that locale.  For example, in example 1, clicking on
+locale 0 will produce a window that looks like:
+</p>
+<img src="E1-L0.png" style="margin:10px; border: 2px solid black;">
+<p>
+(Note: There is overhead generated in tasks, CPU time and clock
+time for the Visual Debug function calls.  <em>chplvis</em> removes
+from the task count the overhead tasks but it can not remove the
+CPU and clock time overhead.)
+</p>
+<p>
+Clicking on the red part of the line between locale 0 and locale 1
+will produce a window that looks like:
+</p>
+<img src="E1-C1.0.png" style="margin:10px; border: 2px solid black;">
+<p>
+It is important to notice the direction of the "arrow" in the header
+for the windows.   This is for communication from locale 1 to locale 0.
+The total communication was 12.  It is further broken out into three
+components:
+<ul>
+  <li> <em>Gets:</em> This is a communication initiated by locale 0 to
+get a data located on locale 1.</li>
+  <li> <em>Puts:</em> This is a communication initiated by <em>locale 1</em>
+to put data from locale 1 onto locale 0.</li>
+  <li> <em>Forks:</em> This where <em>locale 1</em> starts a task running
+on locale 0.  As part of the task start, a block of data is sent to locale 0
+as an argument to the task.  This data is considered a communication
+by <em>chplvis</em></li>
+</ul>
+
+<h4>Example 2</h4>
+In many programs, one will want to look at a number of small parts of
+their program in addition to seeing the total statistics.  prog2.chpl
+gives an example of using the <em>VisualDebug</em> functions
+<em>tagVdebug(name)</em> and <em>pauseVdebug()</em>.
+<pre style="margin:20px">
+// Example 2 of use of VisualDebug module and chplvis tool.
+
+use BlockDist;
+use VisualDebug;
+
+config var ncells = 10;
+
+proc main() {
+
+   // Create a couple of domains and a block mapped data array.
+   const Domain = { 1 .. ncells };
+   const mapDomain = Domain dmapped Block(Domain);
+
+   var  data : [mapDomain] int = 1;
+
+   // Start VisualDebug here
+   startVdebug ("E2");
+
+   // First computation step ... a simple forall
+   forall i in Domain do data[i] += here.id + 1;
+
+   // Write the result, we want to see the results of the above
+   // so we tag before we continue.
+   tagVdebug("writeln 1");
+   writeln("data= ", data);
+
+   // Second computation step ... using the distributed domain
+   tagVdebug("step 2");
+   forall i in mapDomain do data[i] += here.id+1;
+
+   // Don't capture for the writeln
+   pauseVdebug();
+   writeln("data2= ", data);
+
+   // Reduction step
+   tagVdebug("reduce");
+   var i = + reduce data;
+
+   // done with visual debug
+   stopVdebug();
+
+   writeln ("sum is " + i + ".");
+}
+</pre>
+<p>
+Note that the <em>startVdebug("E2");</em> is places after the declarations
```

Comment:
Typo on "places"?  I think it should be placed.  I would also expand "the startVdebug();" to "the startVdebug() call".